### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by FredGenius** [jneilliii/OctoPrint-BambuPrinter#44]
I've installed the plugin, but I can't figure out what to do next - my printer is a P1S.

**Comment by jneilliii** [comment]
Look in settings for the Bambu Printer section in plugins to setup the connection to your printer.

**Comment by FredGenius** [comment]
Thanks, I just found it. Maybe add this too readme?

 

Fred Ö¿Ö

**Comment by jneilliii** [comment]
I don't usually do documentation until I publish a plugin to the official repo. The fact you found this plugin means you are now part of the beta program...good luck.

**Comment by jneilliii** [comment]
I recommend you read through past issues to understand some of the quirks related to the plugin.

**Comment by FredGenius** [comment]
This is a game-changer! Will be my pleasure to help, thanks.

 

Fred Ö¿Ö

**Comment by FredGenius** [comment]
Thanks, will do.

 

Fred Ö¿Ö

**Comment by FredGenius** [comment]
I have the latest version of OctoPrint-BambuPrinter installed and I see a dramatic improvement – I can see files on the sd card and move the carriage, but when I tried to print a gcode file I got this error… 



And this status message…




As before, not complaining, hope this helps.

Thanks for your hard work, please let me know if there’s anything I can do to speed this along.

 

Fred Ö¿Ö

**Comment by jneilliii** [comment]
sorry, screenshots don't come through on email replies @FredGenius so I have no way of seeing your messages.

**Comment by FredGenius** [comment]
Apologies, error message reads ‘[bold] Suppressed command’, ‘The command T1000 was not sent to the printer:’ ‘[detail] Not queuing T1000, that tool doesn’t  exist according to the printer profile or was reported as invalid by the firmware. Make sure your printer profile is set up correctly.’

The State message reads ‘State: Offline after error Could not write to serial port’

 

Fred Ö¿Ö

**Comment by jneilliii** [comment]
Yeah, that means you uploaded to local storage and not the SD card, or trying to start a print that is stored on the local storage of OctoPrint. The plugin only works as a virtual SD card, so you need to make sure to start the files that are indicated as being on the SD card. 

![image](https://github.com/user-attachments/assets/340a8d21-7e6b-4012-a3df-bcf7d8ec0ff5)

I personally filter the file list to only show the SD card files. 

![image](https://github.com/user-attachments/assets/e0c73cb9-ba6e-4095-b45a-eedb460b8e97)

**Comment by guysoft** [comment]
Does the plugin require lan only mode?

**Comment by jneilliii** [comment]
It didn't use to, but I haven't tested the cloud mode in quite some time and Bambu could be blocking its access.
